ush/config_defaults.yaml

```diff
@@ -2811,63 +2107,27 @@ verification:
-  #
-  # RUN_TASKS_METVX_ENS:
-  # Flag that specifies whether to run ensemble verification.  The ensemble
-  # forecast output on which vx will be run may be staged or generated by
-  # running an ensemble of forecasts with the weather model as part of the
-  # current SRW-App-generated experiment.
-  #
-  RUN_TASKS_METVX_DET: False
-  RUN_TASKS_METVX_ENS: False
-  #
   # VX_FCST_MODEL_NAME:
   # String that specifies a descriptive name for the model being verified.
   # This is used in forming the names of the verification output files as
   # well as in the contents of those files.
   #
-  # VX_FIELDS:
-  # The fields or groups of fields on which to run verification.
-  #
-  # VX_APCP_ACCUMS_HH:
-  # The 2-digit accumulation periods (in units of hours) to consider for
-  # APCP (accumulated precipitation).  If VX_FIELDS contains "APCP", then
-  # VX_APCP_ACCUMS_HH must contain at least one element.  If not,
-  # VX_APCP_ACCUMS_HH will be ignored.
-  #
   VX_FCST_MODEL_NAME: '{{ nco.NET }}.{{ task_run_post.POST_OUTPUT_DOMAIN_NAME }}'
-  VX_FIELDS: [ "APCP", "REFC", "RETOP", "SFC", "UPA" ]
-  VX_APCP_ACCUMS_HH: [ "01", "03", "06", "24" ]
   #
-  # VX_FCST_INPUT_BASEDIR:
-  # Location of top-level directory containing forecast (but not obs) files
-  # that will be used as input into METplus for verification.  If not
-  # specified, this gets set to EXPTDIR.
+  # VX_FCST_INPUT_DIR:
+  # Directory template for the path containing forecast (but not obs) files
+  # that will be used as input into METplus for verification.
   #
   # VX_OUTPUT_BASEDIR:
   # Top-level directory in which METplus will place its output.
   #
-  VX_FCST_INPUT_BASEDIR: '{{ "$COMOUT/../.." if user.RUN_ENVIR == "nco" else ( workflow.EXPTDIR if ((workflow_switches.RUN_TASK_RUN_FCST and task_run_fcst.WRITE_DOPOST) or workflow_switches.RUN_TASK_RUN_POST) ) }}'
-  VX_OUTPUT_BASEDIR: '{{ "$COMOUT/metout" if user.RUN_ENVIR == "nco" else workflow.EXPTDIR }}'
+  VX_FCST_INPUT_DIR: '{% if user.RUN_ENVIR == "nco" %}$COMOUT/../..{% else %}{{ workflow.EXPTDIR }}{% endif %}'
+  VX_OUTPUT_BASEDIR: '{% if user.RUN_ENVIR == "nco" %}$COMOUT/metout{% else %}{{ workflow.EXPTDIR }}{% endif %}'
   #
   # File name and path templates used in the verification tasks.
   #
-  FCST_SUBDIR_TEMPLATE: '{{ "${NET}.{init?fmt=%Y%m%d?shift=-${time_lag}}/{init?fmt=%H?shift=-${time_lag}}" if user.RUN_ENVIR == "nco" else "{init?fmt=%Y%m%d%H?shift=-${time_lag}}${SLASH_ENSMEM_SUBDIR_OR_NULL}/postprd" }}{% raw %}{% endraw %}'
-  FCST_FN_TEMPLATE: '{{ "${NET}.t{init?fmt=%H?shift=-${time_lag}}z" }}{{ "${DOT_ENSMEM_OR_NULL}" if user.RUN_ENVIR == "nco" else "" }}{{ ".prslev.f{lead?fmt=%HHH?shift=${time_lag}}.${POST_OUTPUT_DOMAIN_NAME}.grib2" }}{% raw %}{% endraw %}'
-  FCST_FN_METPROC_TEMPLATE: '{{ "${NET}.t{init?fmt=%H?}z" }}{{ "${DOT_ENSMEM_OR_NULL}" if user.RUN_ENVIR == "nco" else "" }}{{ ".prslev.f{lead?fmt=%HHH?}.${POST_OUTPUT_DOMAIN_NAME}_a${ACCUM_HH}h.nc" }}{% raw %}{% endraw %}'
```

Comment:
@mkavulich Same question for `FCST_FN_METPROC_TEMPLATE` as for `FCST_FN_TEMPLATE` above:  Why remove if-statement for NCO mode?